README.md

```diff
@@ -76,3 +76,18 @@ to the fact they are in a responsive grid.
 1. Logos should be uploaded to s3 bucket
 2. Logos should 120px by 40px ( or same aspect )
 3. Logos should be .png
+
+# Deployment
+Single Sign On Dashboard (SSO Dashboard) runs in the AWS IAM account (320464205386) inside the production EKS cluster. 
+
+Currently the application is deployed into 2 different environments: dev and prod, each one running in the correspondent Kubernetes namespaces. 
```

Comment:
There is also infrastructure deployed in the `infosec-prod` AWS account with cross account roles setup between the two accounts. More information on this can be found in https://jira.mozilla.com/browse/IAM-636 . @andrewkrug has mentioned to me what this is that's in `infosec-prod` and why it needs to be there and what remains in `infosec-prod` but I can't find my notes on it for the life of me.

Reply:
thanks for pointing this out. 
Looking further into it, I found that the SSO production container running is assuming the next role: `arn:aws:iam::3*********1:role/SSO-Dashboard-Role` in what I believe is the `infosec-prod` account.
The `stage` deployment is assuming a similar role in what I think is `infosec-dev` account.

Furthermore I found a very clear document from @andrewkrug https://github.com/mozilla-iam/iam-infra/blob/74a68749db6f9043bdd36970d0e94de322cd9804/docs/runbooks/sso-dashboard.md which seems to be slightly out of date because it mentions him as the owner, references the MOC team and some other minor things.
However the information regarding how the application is deployed and which resources are used seem to be accurate.

I'm going to modify this PR removing the part where I mention only one AWS account, and link this document.
Also, as now we are planning for H2, would be a good moment to allocated time to pay the technical debt here and merge the resources spawned in three accounts into aws-iam account, as the document suggests.

Reply:
done.

Reply:
> Also, as now we are planning for H2, would be a good moment to allocated time to pay the technical debt here and merge the resources spawned in three accounts into aws-iam account, as the document suggests.

Reply:
I'll bring this to discussion during our H2 planning. Anyway I'd need your help as the resources to migrate are living in infosec-* AWS accounts.

Reply:
> Anyway I'd need your help as the resources to migrate are living in infosec-* AWS accounts.

Absolutely

Reply:
I've logged SE-1056 and probably we will work on it during H2.
Can this PR be merged now?